Lib/ssl.py

```diff
@@ -780,6 +785,23 @@ def version(self):
     def verify_client_post_handshake(self):
         return self._sslobj.verify_client_post_handshake()
 
+    def key_update(self, updatetype):
```

Comment:
With TLS 1.3, you can also force an immediate rekey. From the documentation, https://www.openssl.org/docs/man1.1.1/man3/SSL_key_update.html

> Alternatively SSL_do_handshake() can be called to force the update to take place immediately.

Reply:
Yeah I also found that during writing docs,

> Alternatively do_handshake()  can be called to force the update to take place immediately.

Do you think it a better approach to wrap OpenSSL API into high level API like this:

```python
    def key_update(self, updatetype, *, deferred=False):
        self._sslobj.key_update(updatetype)
        if not deferred:
            self._sslobj.do_handshake()
```

Or stay with OpenSSL-style API and document the details? Now I prefer the high-level one, leaving the low-level API with the `_ssl` module.